front/src/components/store/DataDetail/DataDetail.vue

```diff
@@ -333,6 +333,9 @@ export default {
     'validation-provider': Vue.extend(ValidationProvider),
   },
   props: {
+    blockId: {
+      type: String,
```

Comment:
Don't you think we need to set a default value ?

I don't know if the prop is not set, what's the value of `blockId` ? `null` ? `undefined` ? empty string ?

Reply:
Finally, prop `blockId` is required